2025-02-17 02:38:02,386 - WARNING - Retrying (Retry(total=4, connect=None, read=None, redirect=None, status=None)) after connection broken by 'ReadTimeoutError("HTTPSConnectionPool(host='quests-usage-dev.prod.zettablock.com', port=443): Read timed out. (read timeout=15)")': /api/user/0x6646de28934127ba20ea5444206cfd1c382a7c17/stats
2025-02-17 02:38:19,170 - WARNING - Retrying (Retry(total=3, connect=None, read=None, redirect=None, status=None)) after connection broken by 'ReadTimeoutError("HTTPSConnectionPool(host='quests-usage-dev.prod.zettablock.com', port=443): Read timed out. (read timeout=15)")': /api/user/0x6646de28934127ba20ea5444206cfd1c382a7c17/stats
2025-02-17 02:38:37,094 - WARNING - Retrying (Retry(total=2, connect=None, read=None, redirect=None, status=None)) after connection broken by 'ReadTimeoutError("HTTPSConnectionPool(host='quests-usage-dev.prod.zettablock.com', port=443): Read timed out. (read timeout=15)")': /api/user/0x6646de28934127ba20ea5444206cfd1c382a7c17/stats
2025-02-17 02:38:56,990 - WARNING - Retrying (Retry(total=1, connect=None, read=None, redirect=None, status=None)) after connection broken by 'ReadTimeoutError("HTTPSConnectionPool(host='quests-usage-dev.prod.zettablock.com', port=443): Read timed out. (read timeout=15)")': /api/user/0x6646de28934127ba20ea5444206cfd1c382a7c17/stats
2025-02-17 02:39:20,760 - WARNING - Retrying (Retry(total=0, connect=None, read=None, redirect=None, status=None)) after connection broken by 'ReadTimeoutError("HTTPSConnectionPool(host='quests-usage-dev.prod.zettablock.com', port=443): Read timed out. (read timeout=15)")': /api/user/0x6646de28934127ba20ea5444206cfd1c382a7c17/stats
2025-02-17 02:39:57,447 - WARNING - Retrying (Retry(total=4, connect=None, read=None, redirect=None, status=None)) after connection broken by 'ReadTimeoutError("HTTPSConnectionPool(host='quests-usage-dev.prod.zettablock.com', port=443): Read timed out. (read timeout=15)")': /api/user/0x6646de28934127ba20ea5444206cfd1c382a7c17/stats
2025-02-17 02:40:14,359 - WARNING - Retrying (Retry(total=3, connect=None, read=None, redirect=None, status=None)) after connection broken by 'ReadTimeoutError("HTTPSConnectionPool(host='quests-usage-dev.prod.zettablock.com', port=443): Read timed out. (read timeout=15)")': /api/user/0x6646de28934127ba20ea5444206cfd1c382a7c17/stats
2025-02-17 02:40:32,274 - WARNING - Retrying (Retry(total=2, connect=None, read=None, redirect=None, status=None)) after connection broken by 'ReadTimeoutError("HTTPSConnectionPool(host='quests-usage-dev.prod.zettablock.com', port=443): Read timed out. (read timeout=15)")': /api/user/0x6646de28934127ba20ea5444206cfd1c382a7c17/stats
2025-02-17 02:40:52,170 - WARNING - Retrying (Retry(total=1, connect=None, read=None, redirect=None, status=None)) after connection broken by 'ReadTimeoutError("HTTPSConnectionPool(host='quests-usage-dev.prod.zettablock.com', port=443): Read timed out. (read timeout=15)")': /api/user/0x6646de28934127ba20ea5444206cfd1c382a7c17/stats
2025-02-17 02:41:16,084 - WARNING - Retrying (Retry(total=0, connect=None, read=None, redirect=None, status=None)) after connection broken by 'ReadTimeoutError("HTTPSConnectionPool(host='quests-usage-dev.prod.zettablock.com', port=443): Read timed out. (read timeout=15)")': /api/user/0x6646de28934127ba20ea5444206cfd1c382a7c17/stats
2025-02-17 02:41:52,680 - WARNING - Retrying (Retry(total=4, connect=None, read=None, redirect=None, status=None)) after connection broken by 'ReadTimeoutError("HTTPSConnectionPool(host='quests-usage-dev.prod.zettablock.com', port=443): Read timed out. (read timeout=15)")': /api/user/0x6646de28934127ba20ea5444206cfd1c382a7c17/stats
2025-02-17 02:42:09,580 - WARNING - Retrying (Retry(total=3, connect=None, read=None, redirect=None, status=None)) after connection broken by 'ReadTimeoutError("HTTPSConnectionPool(host='quests-usage-dev.prod.zettablock.com', port=443): Read timed out. (read timeout=15)")': /api/user/0x6646de28934127ba20ea5444206cfd1c382a7c17/stats
2025-02-17 02:42:27,331 - WARNING - Retrying (Retry(total=2, connect=None, read=None, redirect=None, status=None)) after connection broken by 'ReadTimeoutError("HTTPSConnectionPool(host='quests-usage-dev.prod.zettablock.com', port=443): Read timed out. (read timeout=15)")': /api/user/0x6646de28934127ba20ea5444206cfd1c382a7c17/stats
2025-02-17 02:42:47,095 - WARNING - Retrying (Retry(total=1, connect=None, read=None, redirect=None, status=None)) after connection broken by 'ReadTimeoutError("HTTPSConnectionPool(host='quests-usage-dev.prod.zettablock.com', port=443): Read timed out. (read timeout=15)")': /api/user/0x6646de28934127ba20ea5444206cfd1c382a7c17/stats
2025-02-17 02:43:18,621 - WARNING - Retrying (Retry(total=4, connect=None, read=None, redirect=None, status=None)) after connection broken by 'ReadTimeoutError("HTTPSConnectionPool(host='quests-usage-dev.prod.zettablock.com', port=443): Read timed out. (read timeout=15)")': /api/user/0x6646de28934127ba20ea5444206cfd1c382a7c17/stats
2025-02-17 02:43:35,354 - WARNING - Retrying (Retry(total=3, connect=None, read=None, redirect=None, status=None)) after connection broken by 'ReadTimeoutError("HTTPSConnectionPool(host='quests-usage-dev.prod.zettablock.com', port=443): Read timed out. (read timeout=15)")': /api/user/0x6646de28934127ba20ea5444206cfd1c382a7c17/stats
2025-02-17 02:43:53,089 - WARNING - Retrying (Retry(total=2, connect=None, read=None, redirect=None, status=None)) after connection broken by 'ReadTimeoutError("HTTPSConnectionPool(host='quests-usage-dev.prod.zettablock.com', port=443): Read timed out. (read timeout=15)")': /api/user/0x6646de28934127ba20ea5444206cfd1c382a7c17/stats
2025-02-17 02:44:12,995 - WARNING - Retrying (Retry(total=1, connect=None, read=None, redirect=None, status=None)) after connection broken by 'ReadTimeoutError("HTTPSConnectionPool(host='quests-usage-dev.prod.zettablock.com', port=443): Read timed out. (read timeout=15)")': /api/user/0x6646de28934127ba20ea5444206cfd1c382a7c17/stats
2025-02-17 02:44:36,762 - WARNING - Retrying (Retry(total=0, connect=None, read=None, redirect=None, status=None)) after connection broken by 'ReadTimeoutError("HTTPSConnectionPool(host='quests-usage-dev.prod.zettablock.com', port=443): Read timed out. (read timeout=15)")': /api/user/0x6646de28934127ba20ea5444206cfd1c382a7c17/stats
2025-02-17 02:45:13,559 - WARNING - Retrying (Retry(total=4, connect=None, read=None, redirect=None, status=None)) after connection broken by 'ReadTimeoutError("HTTPSConnectionPool(host='quests-usage-dev.prod.zettablock.com', port=443): Read timed out. (read timeout=15)")': /api/user/0x6646de28934127ba20ea5444206cfd1c382a7c17/stats
2025-02-17 02:45:30,455 - WARNING - Retrying (Retry(total=3, connect=None, read=None, redirect=None, status=None)) after connection broken by 'ReadTimeoutError("HTTPSConnectionPool(host='quests-usage-dev.prod.zettablock.com', port=443): Read timed out. (read timeout=15)")': /api/user/0x6646de28934127ba20ea5444206cfd1c382a7c17/stats
2025-02-17 02:45:48,200 - WARNING - Retrying (Retry(total=2, connect=None, read=None, redirect=None, status=None)) after connection broken by 'ReadTimeoutError("HTTPSConnectionPool(host='quests-usage-dev.prod.zettablock.com', port=443): Read timed out. (read timeout=15)")': /api/user/0x6646de28934127ba20ea5444206cfd1c382a7c17/stats
2025-02-17 02:46:08,130 - WARNING - Retrying (Retry(total=1, connect=None, read=None, redirect=None, status=None)) after connection broken by 'ReadTimeoutError("HTTPSConnectionPool(host='quests-usage-dev.prod.zettablock.com', port=443): Read timed out. (read timeout=15)")': /api/user/0x6646de28934127ba20ea5444206cfd1c382a7c17/stats
2025-02-17 02:46:32,029 - WARNING - Retrying (Retry(total=0, connect=None, read=None, redirect=None, status=None)) after connection broken by 'ReadTimeoutError("HTTPSConnectionPool(host='quests-usage-dev.prod.zettablock.com', port=443): Read timed out. (read timeout=15)")': /api/user/0x6646de28934127ba20ea5444206cfd1c382a7c17/stats
2025-02-17 02:47:08,614 - WARNING - Retrying (Retry(total=4, connect=None, read=None, redirect=None, status=None)) after connection broken by 'ReadTimeoutError("HTTPSConnectionPool(host='quests-usage-dev.prod.zettablock.com', port=443): Read timed out. (read timeout=15)")': /api/user/0x6646de28934127ba20ea5444206cfd1c382a7c17/stats
2025-02-17 02:47:25,516 - WARNING - Retrying (Retry(total=3, connect=None, read=None, redirect=None, status=None)) after connection broken by 'ReadTimeoutError("HTTPSConnectionPool(host='quests-usage-dev.prod.zettablock.com', port=443): Read timed out. (read timeout=15)")': /api/user/0x6646de28934127ba20ea5444206cfd1c382a7c17/stats
2025-02-17 02:47:43,437 - WARNING - Retrying (Retry(total=2, connect=None, read=None, redirect=None, status=None)) after connection broken by 'ReadTimeoutError("HTTPSConnectionPool(host='quests-usage-dev.prod.zettablock.com', port=443): Read timed out. (read timeout=15)")': /api/user/0x6646de28934127ba20ea5444206cfd1c382a7c17/stats
2025-02-17 02:48:03,344 - WARNING - Retrying (Retry(total=1, connect=None, read=None, redirect=None, status=None)) after connection broken by 'ReadTimeoutError("HTTPSConnectionPool(host='quests-usage-dev.prod.zettablock.com', port=443): Read timed out. (read timeout=15)")': /api/user/0x6646de28934127ba20ea5444206cfd1c382a7c17/stats
2025-02-17 02:48:27,317 - WARNING - Retrying (Retry(total=0, connect=None, read=None, redirect=None, status=None)) after connection broken by 'ReadTimeoutError("HTTPSConnectionPool(host='quests-usage-dev.prod.zettablock.com', port=443): Read timed out. (read timeout=15)")': /api/user/0x6646de28934127ba20ea5444206cfd1c382a7c17/stats
